lua/entities/gmod_wire_expression2/core/files.lua

```diff
@@ -6,10 +6,10 @@
 local cv_max_transfer_size = CreateConVar("wire_expression2_file_max_size", "300", { FCVAR_REPLICATED, FCVAR_ARCHIVE }, "Maximum file size in kibibytes.")
```

Comment:
This can be increased to a 16 megabytes or something

Reply:
I am curious of how users would make use of 16MB. 1MB is probably already more than enough.

Reply:
I have this set to 8192 on my server so that E2 raytracers can save images.  Those suckers can get big.